lib/manageiq/api/common/open_api/docs.rb

```diff
@@ -8,7 +8,7 @@ module Common
       module OpenApi
         class Docs
           def self.instance
-            @instance ||= new(Dir.glob(Rails.root.join("public", "doc", "openapi*.json")))
+            @instance ||= new(Dir.glob(Rails.root.join("public", "**", "openapi*.json")))
```

Comment:
Where are other projects storing this file?  I thought we copied the path from catalog

Reply:
catalog is in public/catalog/v1.0.0/  whereas topo and sources is just public/doc/

Reply:
Instead of the open ended **, the other option is to require all consumers to have their openapi docs in public/doc/, but not sure what other constraints this would impose.

Reply:
@bdunne @abellotti my understanding around our `/public/catalog/v1.0.0/` path was to come inline with an IPP doc.

Reply:
I'm not sure who, but someone has run an audit against all of these, so there must be some standardized path defined.

Reply:
I imagine the confusion might have came about because of this wording:
> For consistency and to support automated testing of APIs, all applications MUST serve the current OpenAPI specification in JSON format on $PATH_PREFIX/$APP/v$VERSION/openapi.json.

However, all the APIs do serve the openapi.json on that path, but the file lives in a different location on disk.

Catalog and Approval use a versioned folder that contains the `openapi.json` file:
https://github.com/ManageIQ/approval-api/tree/master/public/approval/v1.0

Topology adds versioning to the openapi.json filename  (`openapi-3-v#.#.#.json`): 
https://github.com/ManageIQ/topological_inventory-api/tree/master/public/doc